src/component/header/Header.js

```diff
@@ -1,30 +1,41 @@
 import PropTypes from 'prop-types';
-
 import {FaSearch} from "react-icons/fa";
 
 import * as hs from './HeaderStyle';
-import {HeaderWrapper, ListSelection} from "./HeaderStyle";
+import OptionList from "../optionList/OptionList";
+import {useCallback, useState} from "react";
 
 function Header({searchOption}) {
 
+  const [searchOptions, setSearchOptions] = useState({
+    startDate: '',
+    endDate: '',
+    status: '',
+  });
+
+  const handleChange = useCallback((name, {target: {value}}) => {
+    setSearchOptions({...searchOptions, [name]: value})
+  }, [searchOptions]);
+
+  const handleSearch = () => {
+    console.log(searchOptions);
+  }
+
   return (
     <hs.Header>
-      <HeaderWrapper>
+      <hs.HeaderWrapper>
         <hs.DateBox>
-          <hs.DateSelection name="startDate"/>
+          <hs.DateSelection onChange={(e) => handleChange("startDate", e)}/>
```

Comment:
```suggestion
          <hs.DateSelection onChange={(e) => handleChange("startDate", e.target.value)}/>
```
그러면 여기도 이런식으로 바꿔야겠네요.
순서가 신경쓰인다면 { name:"startDate",value:e.target.value } 처럼 객체로 감싸서 넘겨도 좋을 것 같습니다.

Reply:
```suggestion
          <hs.DateSelection onChange={(e) => {handleChange("startDate", e)}}/>
```
아 그리고 화살표 함수에서 () => ~~ 하면 ~~가 바로 return 된다는 뜻입니다. 현재 handleChange에는 리턴 값이 없으니 한번 감싸 주는게 맞겠군요

Reply:
화살표 함수를 그냥 사용해도 되길래 {} 넣을 생각은 한번도 안해봤네. 확실히 {}를 추가하는 것이 의도에 맞는거 같아! 반환값이 없는 모든 화살표 함수에 {}를 추가할게